setup/web_server_configuration.rst

```diff
@@ -287,6 +287,13 @@ The **minimum configuration** to get your application running under Nginx is:
             # try to serve file directly, fallback to index.php
             try_files $uri /index.php$is_args$args;
         }
+        
+        # optionally disable falling back to php script for the asset directories
+        # which will allow nginx to return a 404 error when files are
+        # not found instead of passing the request to Symfony
+        location /bundles {
+            try_files $uri $uri/ =404;
```

Comment:
Actually, if you are looking for a file, the `$uri/` lookup seems useless to me.